File: scratch/scratch_read_logs.js
const Database = require('better-sqlite3');
const path = require('path');

const dbPath = path.join(__dirname, '../kiwoom.db');
console.log('Opening DB:', dbPath);

try {
    const db = new Database(dbPath, { readonly: true });
    
    // 테이블 목록 확인
    const tables = db.prepare("SELECT name FROM sqlite_master WHERE type='table'").all();
    console.log('Tables in DB:', tables.map(t => t.name).join(', '));
    
    if (tables.some(t => t.name === 'ai_execution_logs')) {
        console.log('\n--- Recent 20 AI Execution Logs ---');
        const logs = db.prepare(`
            SELECT id, agentId, agentName, triggerType, targetType, status, queuedAt, modelName, error
            FROM ai_execution_logs 
            ORDER BY queuedAt DESC 
            LIMIT 20
        `).all();
        console.table(logs);
    } else {
        console.log('Table ai_execution_logs does not exist.');
    }
} catch (err) {
    console.error('Error reading DB:', err.message);
}

File: scratch/scratch_read_logs.py
import sqlite3
import os

db_path = os.path.join(os.path.dirname(__file__), '../kiwoom.db')
print('Opening DB:', db_path)

try:
    conn = sqlite3.connect(db_path)
    cursor = conn.cursor()
    
    # 테이블 목록 확인
    cursor.execute("SELECT name FROM sqlite_master WHERE type='table'")
    tables = [t[0] for t in cursor.fetchall()]
    print('Tables in DB:', ', '.join(tables))
    
    if 'ai_execution_logs' in tables:
        print('\n--- Recent 20 AI Execution Logs ---')
        cursor.execute("""
            SELECT id, agentId, agentName, triggerType, targetType, status, queuedAt, modelName, error 
            FROM ai_execution_logs 
            ORDER BY queuedAt DESC 
            LIMIT 20
        """)
        rows = cursor.fetchall()
        
        # 프린트 형식
        col_names = [d[0] for d in cursor.description]
        print(f"{' | '.join(col_names)}")
        print("-" * 120)
        for row in rows:
            print(" | ".join(str(val) for val in row))
    else:
        print('Table ai_execution_logs does not exist.')
        
    conn.close()
except Exception as e:
    print('Error reading DB:', str(e))
